### ISSUE TITLE
Hide FirstJoin Meassage - happens with cmi.messages.disablelogin?

### ISSUE THREAD

**Issue by CloudeLecaw** [Zrips/CMI#5248]
We want to hide the join meassagefor first joiner

What we tried at the moment:

```
Messages:
  Login:
    # If set to true, login message wont be shown
    Disabled: false
    # Defines number of players from which to automatically start hiding join messages
    # Set to -1 to disable this
    AutoHideFrom: -1
    Custom:
      # If set to true, custom login message will be used. cmi.messages.disablelogin can be used to disable message for player
      Use: true
  Logout:
    # If set to true, logout message wont be shown
    Disabled: false
    # Defines number of players from which to automatically start hiding logout messages
    # Set to -1 to disable this
    AutoHideFrom: -1
    Custom:
      # If set to true, custom logout message will be used. cmi.messages.disablequit can be used to disable message for player
      Use: true
  # Check locale file for translation and custom placeholders: [playername], [totalUsers], [onlinePlayers]
  FirstJoinMessage:
    Use: true
```

and we gave their NewbieGroup the permission.
    - cmi.messages.disablelogin
    - cmi.messages.disablequit

Does the hide permission include the first joiner?

**Comment by Zrips** [comment]
Players who join for the first time will trigger first time join message and not the regular one. Tested this one to be sure and everything seam to work properly. Unless you have another plugin which triggers join message all seams to work correctly at this point. There might be some issues with permissions if you have permission plugin which updates player permissions after CMI checks for one, which could be the case for the first joiners as some new data needs to be generated. But by the code itself, first joiners will set regular join message to empty field to avoid showing it up. So you will need to investigate it at your end to determine cause.

**Comment by CloudeLecaw** [comment]
Thats what i did and seems to work. I cleared out the first join meassage given from cmi to ''
I also gave them the quit/join disable permission.
We had the Problem before that some new players come in and leave and come back etc.
This was annoing players. So we manged now that they dont getninformed. We wished to had an option to be able to purge/remove playerdata of playere with these just 1-2 min playtime. But currently therr seems to be nonsolution for this.
